Detail more precisely default on dashboard (#11220)
traefik/traefik: docs/content/operations/dashboard.md

## Insecure Mode

The dashboard is also enabled when API is in [insecure](./api.md#insecure) mode.
This mode is not recommended and should be used for testing purpose only.

```yaml tab="File (YAML)"
api:
  insecure: true
```

```toml tab="File (TOML)"
[api]
  insecure = true
```

```bash tab="CLI"
--api.insecure=true
```

When _insecure_ mode is enabled, one can access the dashboard on the `traefik` port (default: `8080`) of the Traefik instance,
at the following URL: `http://<Traefik IP>:8080/dashboard/` (trailing slash is mandatory).

## Disable the dashboard

> By default, when API is enabled, dashboard is enabled.
> If needed, the API can be enabled without the dashboard.

Reviewer: ```suggestion
By default, the Dashboard is enabled when the API is enabled.
If necessary, the Dashboard can be disabled by using the following option.
```

## Insecure Mode

When _insecure_ mode is enabled, one can access the dashboard on the `traefik` port (default: `8080`) of the Traefik instance,
at the following URL: `http://<Traefik IP>:8080/dashboard/` (trailing slash is mandatory).

> This mode is **not** recommended because it does not allow security features. For example, it is not possible to add an authentication middleware with this mode.

Reviewer: ```suggestion
This mode is **not** recommended because it does not allow security features.
For example, it is not possible to add an authentication middleware with this mode.
```

## Insecure Mode

When _insecure_ mode is enabled, one can access the dashboard on the `traefik` port (default: `8080`) of the Traefik instance,
at the following URL: `http://<Traefik IP>:8080/dashboard/` (trailing slash is mandatory).

This mode is **not** recommended because:

1. Sensitive data is easily accessible on Traefik network
2. Security features like adding an authentication middleware cannot be used with this mode

It should be used for testing purpose **only**.

To enable the _insecure_ mode, use the following options from [Traefik's API](./api.md#insecure):

```yaml tab="File (YAML)"
api:
  insecure: true
```

```toml tab="File (TOML)"
[api]
  insecure = true
```

```bash tab="CLI"
--api.insecure=true
```

> ## Disable the dashboard

Author: ```suggestion
## Disable the Dashboard
```
